shell/platform/windows/windows_proc_table.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_WINDOWS_PROC_TABLE_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_WINDOWS_PROC_TABLE_H_
+
+#include "flutter/fml/native_library.h"
+
+#include <optional>
+
+namespace flutter {
+
+// Lookup table for Windows APIs that aren't available on all versions of
+// Windows.
+class WindowsProcTable {
+ public:
+  WindowsProcTable();
+  virtual ~WindowsProcTable();
+
+  // Retrieves the pointer type for a specified pointer.
+  //
+  // Used to react differently to touch or pen inputs. Returns zero on failure.
```

Comment:
Returns false on failure.

Reply:
This follows the official docs: https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getpointertype#return-value

Reply:
It's confusing in the current form, since you have the "second" return value in the pointer which is an integer. So it can read as if the returned pointer_type will be zero on failure.

Reply:
Gotcha. Updated!